src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setProperties(colorSupplier.getRandomColor(), random.nextInt(100));
+        return ball;
```

Comment:
```suggestion
        String color = colorSupplier.getRandomColor();
        int number = random.nextInt(100);
        return new Ball(color, number);
```

100 is a magic number, create constant